.github/workflows/pipeline.yml

```diff
@@ -43,3 +43,23 @@ jobs:
     - name: JS Lint
       run: |
         npm run lint:ts
+    - name: Run coders examples
+      run: |
+        npx hardhat run scripts/coders.ts
+    - name: Run converters examples
+      run: |
+        npx hardhat run scripts/converters.ts
+    - name: Run API Client examples
+      env:
+        RPC_URL: ${{ secrets.RPC_URL }}
+        UMB_CHAIN_ADDRESS: ${{ secrets.UMB_CHAIN_ADDRESS }}
+        API_KEY: ${{ secrets.APY_KEY }}
+      run: |
+        npx hardhat run scripts/api-client.ts
+    - name: Run proof verification example
+      env:
+        RPC_URL: ${{ secrets.RPC_URL }}
+        UMB_CHAIN_ADDRESS: ${{ secrets.UMB_CHAIN_ADDRESS }}
+        API_KEY: ${{ secrets.APY_KEY }}
+      run: |
+        npx hardhat run scripts/proof-verification.ts
```

Comment:
lets add also `hardhat test`

Reply:
Perfect, added. Once CI passes I will merge and close the PR :)